detekt.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: "io.gitlab.arturbosch.detekt"
+
+detekt {
+    toolVersion = Versions.toolDetekt
+    // The directories where detekt looks for input files. Defaults to `files("src/main/java", "src/main/kotlin")`.
+    input = files("src/main/java")
+    parallel = true
+    config = files("$rootDir/default-detekt-config.yml")
+    reports {
```

Comment:
Guess you dont need both xml and html reports, so you can choose one reporting format.